src/app/components/game-components/GameComponentUtils.ts

```diff
@@ -22,7 +23,8 @@ export class GameComponentUtils {
                 dy = 0;
                 angle = 180;
                 break;
-            case Orthogonal.RIGHT:
+            default:
+                assert(direction === Orthogonal.RIGHT, 'Direction must be RIGHT');
```

Comment:
"Direction should be one of the 4 orthogonal direction but is not!" ?

Reply:
changé en Utils.defaultCase